src/main/java/uk/gov/justice/digital/hmpps/cmd/api/service/UserPreferenceService.kt

```diff
@@ -37,16 +37,20 @@ class UserPreferenceService(
     repository.save(userPreferences)
   }
 
```

Comment:
Problem was it was calling getOrCreate.. from the batch job for a non-CMD user, when it should be just calling a get fn